src/group_theory/subgroup.lean

```diff
@@ -970,6 +973,20 @@ le_antisymm
   (infi_le_of_le (normal_closure s) (infi_le_of_le (by apply_instance)
     (infi_le_of_le subset_normal_closure (le_refl _))))
 
+theorem normal_closure_eq_self_of_normal {H : subgroup G} [H.normal] : normal_closure ↑H = H :=
```

Comment:
```suggestion
@[simp] theorem normal_closure_eq_self {H : subgroup G} [H.normal] : normal_closure ↑H = H :=
```
we typically don't include typeclass arguments in the name, unless the resulting name becomes very ambiguous. With my proposed name, most users will still guess that the `[normal]` assumption must be there.

Reply:
Fine with me.